### PR TITLE
Add predefined container images

### PR DESCRIPTION
**Pull request by dvonessen**
What does this PR change? What problem does it solve?
-----------------------------------------------------
Added a new class which is used as a container for predefined images. 
Those images can be used within gcip but also in all jobs if wished.

Was the change discussed in an issue?
------------------------------------------------------------

Checklist
---------

- [x] I have read the [Contribution Guidelines](https://github.com/dbsystel/gitlab-ci-python-library/blob/main/CONTRIBUTING.md)
- [x] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
- [ ] I have added tests for all changes in this PR
- [x] I have added api documentation for the changes
- [x] I added a changelog entry to the unreleased section with the PR number. [Changelog](https://github.com/dbsystel/gitlab-ci-python-library/blob/main/CHANGELOG.md)
- [x] Linting is done, and linting jobs are successfull
- [x] I'm done, this Pull Request is ready for review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +30,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   a dummy string is returned instead of raising a KeyError.
 * The `gcip.Pipeline` has now an `add_services()` method instead `add_service()` allowing to pass multiple
   services at once.
+* Now jobs which hardcode images, now using PredefinedImages images instead.
```
Comment (ThomasSteinbach) [inline CHANGELOG.md:33]: T9 failure ^^
```suggestion
* Now jobs with hardcode images, now using PredefinedImages images instead.
```

File: gcip/addons/container/images.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+class PredefinedImages():
+    """
+    PredefinedImages provides container images objects that are widley used withing the `gcip`.
+    """
+    KANIKO: Image = Image("gcr.io/kaniko-project/executor:debug", entrypoint=[""])
+    CRANE: Image = Image("gcr.io/go-containerregistry/crane:latest")
+    DIVE: Image = Image("wagoodman/dive:latest")
+    GCIP: Image = Image("thomass/gcip:0.3.0")
```
Comment (ThomasSteinbach) [inline gcip/addons/container/images.py:11]: we should build 'latest' too. tbd

**Review by ThomasSteinbach — APPROVED** [review]

File: gcip/addons/container/images.py (hunk 2 above)
Comment (dvonessen) [reply gcip/addons/container/images.py:11]: Added issue #39

File: CHANGELOG.md (hunk 1 above)
Comment (dvonessen) [reply CHANGELOG.md]: Done

**Pull request merged**